x-pack/plugins/infra/public/containers/logs/log_source/log_source.ts

```diff
@@ -194,21 +184,23 @@ export const useLogSource = ({
     isUninitialized,
     derivedIndexPattern,
     // Failure states
+    hasFailedLoading,
     hasFailedLoadingSource,
     hasFailedLoadingSourceStatus,
-    loadSourceFailureMessage,
+    hasFailedResolvingSource,
+    latestLoadSourceFailures,
```

Comment:
This can now represent multiple concurrent failures. The canonical case for this is a broken KIP reference in which case the resolution and the status check likely fail in parallel.